ngraph/test/runtime/opset0_tbl.hpp

```diff
@@ -70,6 +70,7 @@ NGRAPH_OP(ConvolutionBackpropData, ngraph::op::v0)
 NGRAPH_OP(Cos, ngraph::op)
 NGRAPH_OP(Cosh, ngraph::op)
 NGRAPH_OP(CumSum, ngraph::op::v0)
+NGRAPH_OP(CTCGreedyDecoder, ngraph::op::v0)
```

Comment:
`opset0` is quite confusing here, what do you think about adding this op straight to [opset_int_tbl.hpp](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/runtime/interpreter/opset_int_tbl.hpp#L21)?

Reply:
I'm not so sure, it's not clear for me right now how should I decide where to put it, there are only a few ops in that file and I don't understand what decides where the op is placed. If there is no clear criteria I'd prefer to put them all in the same place.

Reply:
CTCGreedyDecoder should be added to opset 1 to align with documentation.